modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/TableKeyDetails/index.js

```diff
@@ -0,0 +1,308 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import getCurrentEndDate from '../../../../common/utils/getCurrentEndDate';
+import {useCustomerPortal} from '../../context';
+import {getPascalCase} from '../../utils/getPascalCase';
+
+const HOST_NAME = 'Host Name';
+const IP_ADDRESSES = 'IP Addresses';
+const MAC_ADDRESSES = 'Mac Addresses';
+
+const TableKeyDetails = ({
+	ACTIVATION_STATUS,
+	activationKeys,
+	setIsCopyTextToClipboard,
+	setValueToCopyToClipboard,
+}) => {
+	const [{assetsPath}] = useCustomerPortal();
+	const SUBSCRIPTION_IMAGE_FILE = 'dxp_icon.svg';
+
+	const instanceSizeFormated = activationKeys.sizing.slice(7, 8);
+	const productNameFormated = activationKeys.productName.slice(0, 4);
+	const productName = getPascalCase(activationKeys.licenseEntryType)
+		.split('-')
+		.join(' ');
```

Comment:
I think you can use .replace here

`.replace('-', ' ')`

Reply:
You're right! Thank you, Igor